text/conversations/companions/companion_cv_eder_quest.stringtable

```diff
@@ -65,14 +65,14 @@
     </Entry>
     <Entry>
       <ID>13</ID>
-      <DefaultText>"Ich glaube, weil ich dich nicht kannte. Es ist nicht leicht, einem Fremden zu vertrauen, der von sich behauptet, ein Wächter zu sein."</DefaultText>
-      <FemaleText />
+      <DefaultText>"Ich schätze, weil ich dich nicht kannte. Es ist nicht leicht, einem Fremden zu vertrauen, der von sich behauptet, ein Wächter zu sein."</DefaultText>
+      <FemaleText>"Ich schätze, weil ich dich nicht kannte. Es ist nicht leicht, einer Fremden zu vertrauen, die von sich behauptet, eine Wächterin zu sein"</FemaleText>
     </Entry>
     <Entry>
       <ID>14</ID>
-      <DefaultText>"Natürlich kannte ich tief in meinem Inneren die Wahrheit. Ein Wächter kann keine Gedanken lesen."
+      <DefaultText>"Natürlich kannte ich die Wahrheit, tief in meinem Inneren . Ein Wächter kann keine Gedanken lesen."
```

Comment:
Haha, oh Mann...in dem Fall hast du natürlich Recht.

So viel Text für die Katz... :dancers: